modules/sandboxed-containers-dashboard.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-dashboard_{context}"]
+= Viewing the {sandboxed-containers-first} dashboard
+
+You can access the {sandboxed-containers-first} dashboard in the Metrics UI in the web console.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed.
+* You have {sandboxed-containers-first} installed.
+* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects.
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Observe* → *Dashboards*.
+
+. From the *Dashboard* drop-down list, select the *Sandboxed Containers* dashboard.
+
+. Optional: Select a time range for the graphs in the *Time Range* list.
+
+* Select a pre-defined time period.
+
+* Set a custom time range by selecting *Custom time range* in the *Time Range* list.
+
+.. Define the date and time range for the data you want to view.
+
+.. Click *Save* to save the custom time range.
+
+. Optional: Select a *Refresh Interval*.
+
+The dashboard is visualized on the page with the following metrics from the Kata guest OS category:
+
+Number of running VMs:: Displays the number of sandboxed containers running on your worker nodes.
```

Comment:
Is that total VMs per cluster or VMs per node. Not clear from this comment.

cc @littlejawa

Reply:
This is the total number of VMs per cluster.

Reply:
Thanks @fgiudici, we should make that clear here.